src/tooltip/tooltip.ts

```diff
@@ -178,18 +156,14 @@ export class NgbTooltip implements OnInit, OnDestroy {
         this._document.querySelector(this.container).appendChild(this._windowRef.location.nativeElement);
       }
 
-      this._windowRef.instance.placement = Array.isArray(this.placement) ? this.placement[0] : this.placement;
+      if (!Array.isArray(this.placement)) {
```

Comment:
Let's open a separate feature PR for this later for ALL widgets supporting positioning